src/wallet/asyncrpcoperation_sendmany.h

```diff
@@ -51,7 +51,7 @@ struct WitnessAnchorData {
 
 class AsyncRPCOperation_sendmany : public AsyncRPCOperation {
 public:
-    AsyncRPCOperation_sendmany(CMutableTransaction contextualTx, std::string fromAddress, std::vector<SendManyRecipient> tOutputs, std::vector<SendManyRecipient> zOutputs, int minDepth, CAmount fee = ASYNC_RPC_OPERATION_DEFAULT_MINERS_FEE, UniValue contextInfo = NullUniValue, bool sendChangeToSource = false);
+    AsyncRPCOperation_sendmany(CMutableTransaction contextualTx, const std::string& fromAddress, std::vector<SendManyRecipient>&& tOutputs, std::vector<SendManyRecipient>&& zOutputs, int minDepth, CAmount fee = ASYNC_RPC_OPERATION_DEFAULT_MINERS_FEE, UniValue contextInfo = NullUniValue, bool sendChangeToSource = false);
     virtual ~AsyncRPCOperation_sendmany();
     
     // We don't want to be copied or moved around
```

Comment:
Checking the list by Codacy, it seems to me also the method:
`std::array<unsigned char, ZC_MEMO_SIZE> get_memo_from_hex_string(const std::string& s);`
at line 103 should be modified (together with its implementation in `asyncrpcoperation_sendmany.cpp`).

Reply:
Done